pkg/kn/commands/namespaced.go

```diff
@@ -0,0 +1,49 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// TODO: default namespace should be same scope for the request
+const defaultNamespace = "default"
+
+// AddNamespaceFlags adds the namespace-related flags:
+// * --namespace
+// * --all-namespaces
+func AddNamespaceFlags(flags *pflag.FlagSet, allowAll bool) {
+	flags.StringP(
+		"namespace",
+		"n",
+		"",
+		"Namespace to use. (default \"default\")",
```

Comment:
IMO, we can re-phrase this to
```
List the requested object(s) in given namespace.
```
makes the help text align with `all-namespaces` option.